### PR TITLE
Updated peoples information

### PR DESCRIPTION
**Pull request by bearecinos**
Issue #106

### CODE REVIEW

**Comment by fmaussion** [comment]
Both David and Lizz changed institutions again haha.

David I'm not sure where he is now, but @ehultee is at Middleburry College

**Comment by fmaussion** [comment]
Can you also add:
- @Keeptg (Li Fei, Institute of Tibetan Plateau Research, Chinese Academy of Sciences) and 
- @sarah-hanus (Sarah Hanus, https://www.geo.uzh.ch/geolean/en/department/Staff/?content=hanussarah)
- @MatCast (University of Innsbruck)

Thanks!

**Comment by ehultee** [comment]
I'm at Middlebury College (one r) 😃 

Possibly also needs a line break between Lilian's and my entry?  In full screen view of the website I appear together with Lilian

This is silly but I was tagged, so here I am 🙃

**Comment by fmaussion** [comment]
> This is silly

Are you saying the OGGM website is silly? :crab:

**Comment by bearecinos** [comment]
Hi the issue with @ehultee name is fixed in this pull request.. will change to the new affiliations and send a new one

**Comment by bearecinos** [comment]
Ah I also added my new paper :) to the publications .. Cheers

File: _data/people.yml [hunk 1]
```diff
@@ -174,3 +174,21 @@
...
+  github: Keeptg
+  visible: true
+
+- first_name: Sarah
+  last_name: Hanus
+  institution: University of Zurich
+  github: sarah-hanus
+  visible: true
+
+- first_name: Mat
```
Comment (fmaussion) [inline _data/people.yml:190]:
```suggestion
- first_name: Matteo
```

File: _data/people.yml [hunk 2]
```diff
@@ -174,3 +174,21 @@
...
+  last_name: Cast
```
Comment (fmaussion) [inline _data/people.yml:191]:
```suggestion
  last_name: Castellani
```

Comment (bearecinos) [reply _data/people.yml:191]: ah you did not send full name 🤔 ... will modify this and commit

Comment (bearecinos) [reply _data/people.yml]: see changes in the commit below

**Pull request merged**